test/functional/ios/crash-specs.js

```diff
@@ -17,7 +17,7 @@ describe('crash recovery @skip-real-device', function () {
       .elementByAccessibilityId("Crash")
       .click()
       .then(function () {
```

Comment:
@jlipps yeah i can do that i had to increase from `10000` to `30000` to make test pass i can try gradually with 15 sec to 20 sec. Comment is a good idea